src/Components/OATGraphViewer/OATGraphViewer.styles.tsx

```diff
@@ -49,29 +54,76 @@ export const getGraphViewerStyles = () => {
         handle: [
             classNames.handle,
             {
+                background: 'transparent !Important',
+                border: '0px !Important',
+                left: '60% !Important',
+                top: '50% !Important'
+            } as IStyle
+        ],
+        componentHandleFocus: [
+            classNames.componentHandleFocus,
+            {
+                left: '20% !Important',
                 background: theme.semanticColors.variantBorder
             } as IStyle
         ],
-        componentHandle: [
-            classNames.componentHandle,
+        componentHandleHidden: [
+            classNames.componentHandleHidden,
+            {
+                left: '60% !Important',
+                top: '50% !Important',
+                background: 'transparent !Important',
+                border: '0px !Important'
+            } as IStyle
+        ],
+        relationshipHandleFocus: [
+            classNames.relationshipHandleFocus,
             {
-                left: '16.5% !Important',
+                left: '45% !Important',
                 background: theme.semanticColors.variantBorder
             } as IStyle
         ],
-        relationshipHandle: [
-            classNames.relationshipHandle,
+        relationshipHandleHidden: [
+            classNames.relationshipHandleHidden,
             {
+                left: '60% !Important',
+                top: '50% !Important',
+                background: 'transparent !Important',
+                border: '0px !Important'
+            } as IStyle
+        ],
+        extendHandleFocus: [
+            classNames.extendHandleFocus,
+            {
+                left: '100% !Important',
                 background: theme.semanticColors.variantBorder
             } as IStyle
         ],
-        extendHandle: [
-            classNames.extendHandle,
+        extendHandleHidden: [
+            classNames.extendHandleHidden,
+            {
+                left: '60% !Important',
+                top: '50% !Important',
+                background: 'transparent !Important',
+                border: '0px !Important'
+            } as IStyle
+        ],
+        untargetRelationshipHandleFocus: [
+            classNames.untargetRelationshipHandleFocus,
             {
-                left: '83.5% !Important',
+                left: '70% !Important',
                 background: theme.semanticColors.variantBorder
             } as IStyle
         ],
+        untargetRelationshipHandleHidden: [
+            classNames.untargetRelationshipHandleHidden,
+            {
+                left: '60% !Important',
+                top: '50% !Important',
+                background: 'transparent !Important',
+                border: '0px !Important'
```

Comment:
Why is important used so heavily through here? Can't we just provide more specific CSS selectors to override the inherited styles?